docs/next.config.js

```diff
@@ -49,16 +50,9 @@ if (reactStrictMode) {
 const isDeployment = process.env.NETLIFY === 'true';
 
 module.exports = withTM({
-  eslint: {
-    ignoreDuringBuilds: true,
-  },
+  ...nextConfig.baseline,
```

Comment:
Would it make sense to do this more plugin style?

```tsx
withTM(withBaseline({
  // next config ...
}))
```

It would allow us to move the responsibility of merging the configs into the plugin.

Reply:
It sounds good. We could almost have a flow-right helper too:

```jsx
flowRight([
  withTM,
  withBaseline,
])({})
```